packages/vite-plugin-cloudflare/src/runner-worker/module-runner.ts

```diff
@@ -77,6 +78,18 @@ export async function createModuleRunner(
 			async runExternalModule(filepath) {
 				const moduleRE = new RegExp(MODULE_PATTERN);
 
+				if (filepath === "cloudflare:workers") {
+					const originalCloudflareWorkersModule = await import(
+						"cloudflare:workers"
+					);
+					return {
+						...originalCloudflareWorkersModule,
+						env: stripInternalEnv(
+							originalCloudflareWorkersModule.env as WrapperEnv
+						),
+					};
+				}
```

Comment:
Do you think it matters that you're returning a normal object rather than a module namespace object here? Not sure if this could lead to behaviour differences.

Reply:
no I don't think it matters, I can't really imagine how this could effect user's code 🤔 do you have something in mind? or a different way of implementing this?

Also even if there could be behavioural differences I can't imagine them being very broad in scope (i.e. I think this could at most introduce some minor bug that would only apply to dev when you access the module, which we could look into and solve when reported), so this feels to me pretty safe overall 🤔

Reply:
I'm happy with that. Alternative would be to use a virtual module but this is nicer because it keeps everything in the runner worker. How about using `Object.freeze()` so the object is still immutable?

```suggestion
					return Object.freeze({
						...originalCloudflareWorkersModule,
						env: stripInternalEnv(
							originalCloudflareWorkersModule.env as WrapperEnv
						),
					});
				}
```

Reply:
> How about using Object.freeze() so the object is still immutable?

Sure 👍, but what about doing `Object.seal()` instead?
[mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import#module_namespace_object) states that the module namespace objects are _sealed_ not _frozen_